src/main/java/Duke.java

```diff
@@ -1,10 +1,64 @@
+import duke.command.Command;
+import duke.exception.DukeException;
+import duke.parser.Parser;
+import duke.storage.Storage;
+import duke.task.TaskList;
+import duke.ui.Ui;
+
+import java.io.FileNotFoundException;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeFormatterBuilder;
+import java.util.*;
+
+/**
+ * Contains main method for Duke chatbot.
+ * In charge of initializing UI and storage with taskList.
+ */
 public class Duke {
+    private static final String filePath = "data/duke.txt";
+
+
+    private Storage storage;
+    private TaskList tasks;
+    private Ui ui;
+
+    /**
+     * Loads the Duke chatbot with a given filePath to the saved file.
+     * If there is no file in the filePath, Storage will create a new file.
+     * @param filePath File's path.
+     */
```

Comment:
The coding standard suggests to add a line break between description and parameter section. This might have been forgotten since it seems that there are line breaks for other comments.